eip712/build.gradle

```diff
@@ -0,0 +1,21 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply from: '../buildsystem/coverageReport.gradle'
+
+sourceCompatibility = JavaVersion.VERSION_1_8
```

Comment:
check if we wanted to change this ;)

Reply:
Will do in https://github.com/gnosis/svalinn-kotlin/pull/32